backend/src/main/resources/org/cryptomator/hub/flyway/V11__Update_Audit_Events.sql

```diff
@@ -3,14 +3,41 @@
 TRUNCATE "audit_event" CASCADE;
 ALTER SEQUENCE "audit_event_id_seq" RESTART;
 
+-- Add and rename columns to/of existing events
```

Comment:
Let's go nuclear and promote the `TRUNCATE` to a `DROP` and use this chance to rename tables, so they have a common prefix `audit_event_*`. What do you think?

Reply:
I don't think it's necessary to promote `TRUNCATE` to `DROP` because we'd probably leave the "parent" table `audit_event` as is, right?

But I think it makes sense to rename the tables to `audit_event_*`, e.g., `audit_event_unlock_vault`, so that the audit event tables are grouped together. Or maybe even `audit_event_vault_unlock` so that "audit events" of "vault" are grouped together (`*_create`, `*_update`, `*_add_member`, `*_remove_member`, `*_grant_access`).

Reply:
Fixed in 0df8420076f892e01e91ddb372e795ac40ee0c63.